docs/design-time-builds.md

```diff
@@ -39,25 +40,25 @@ If you've determined that your target needs to run in a design-time build, using
 ```
 The `AddAdditionalReferences` target will run in both normal builds _and_ design-time builds, leading to consistent results between them.
 
-### Determining whether a target is run in a design-time build
+### Determining whether a target is running in a design-time build
```

Comment:
Me no speak English good, but I prefer the previous version. "Running" to me implies a point in time.

Reply:
A target could run in both normal and design-time builds, but it can only be running in one.